inspectit-oce-core/src/main/java/rocks/inspectit/oce/core/instrumentation/context/InspectitContext.java

```diff
@@ -0,0 +1,168 @@
+package rocks.inspectit.oce.core.instrumentation.context;
+
+import io.grpc.Context;
+import io.opencensus.common.Scope;
+import io.opencensus.tags.*;
+import rocks.inspectit.oce.core.instrumentation.config.model.ResolvedDataProperties;
+import rocks.inspectit.oce.core.tags.CommonTagsManager;
+
+import java.util.*;
+
+public class InspectitContext implements AutoCloseable {
+
+    /**
+     * Package private for testing
+     */
```

Comment:
Guava provides and annotation for this `@VisibleForTesting`. It is doing nothing but just for marking an method that the visibility is wider than required, so you don't have to comment this in the docs.